src/core/meshcop/dataset_manager.hpp

```diff
@@ -154,7 +154,11 @@ class DatasetManager : public InstanceLocator
      * @retval kErrorNoBufs  Insufficient buffer space to send.
      *
      */
-    Error SendSetRequest(const Dataset::Info &aDatasetInfo, const uint8_t *aTlvs, uint8_t aLength);
+    Error SendSetRequest(const Dataset::Info &aDatasetInfo,
+                         const uint8_t *      aTlvs,
+                         uint8_t              aLength,
+                         otMgmtSetCallback    aCallback,
+                         void *               aContext);
```

Comment:
nit: add `@param` for the two new parameters.

Reply:
Done.